api/net/Areas/Subscriber/Controllers/WorkOrderController.cs

```diff
@@ -156,5 +159,52 @@ public async Task<IActionResult> RequestTranscriptionAsync(long contentId)
                 return new JsonResult(new WorkOrderModel(workOrder, content, _serializerOptions));
         }
     }
+
+    /// <summary>
+    /// Request a transcript for the content for the specified 'contentId'.
+    /// Publish message to kafka to request a transcription.
+    /// </summary>
+    /// <param name="contentId"></param>
+    /// <param name="uid"></param>
+    /// <returns></returns>
+    [AllowAnonymous]
+    [HttpGet("transcribe/{contentId}")]
+    [Produces(MediaTypeNames.Application.Json)]
+    [ProducesResponseType((int)HttpStatusCode.OK)]
+    [ProducesResponseType((int)HttpStatusCode.NoContent)]
+    [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
+    [SwaggerOperation(Tags = new[] { "WorkOrder" })]
+    public async Task<IActionResult> RequestAnonymousTranscriptionAsync(long contentId, int uid)
```

Comment:
Ideally the notification link will not go to this endpoint, but rather to the subscriber site.  There is a separate story to create this static page.  Then it will make the anonymous request to the API.